specs/unified_kernel_image.md

```diff
@@ -51,6 +51,7 @@ UKIs consist of the following resources:
 * Optionally, a compiled Devicetree database file, for systems which need it, in the `.dtb` section.
 * Optionally, the public part of a public-private key pair in PEM format used to sign the image, in the `.pcrpkey` section.
 * Optionally, a JSON file encoding expected PCR 11 hash values seen from userspace once the UKI has booted up, along with signatures of these expected PCR 11 hash values, in the `.pcrsig` section. The signatures must also match the abovementioned key pair.
+* Optionally, a CSV file encoding the SBAT identifier for the image, in the `.sbat` section. The [SBAT format is defined by the Shim project](https://github.com/rhboot/shim/blob/main/SBAT.md), and used for UEFI revocation purposes.
```

Comment:
hmm, singular "indentifier"? i mean, it might contain more than one

maybe `s/identifier/revocation metadata/`, i.e. be more vague and avoid singular/plural questions?

Reply:
changed to simply 'metadata' to keep it short an concise, the purpose is described later in the sentence